app/controllers/forms/ContributionRequest.scala

```diff
@@ -28,7 +28,9 @@ case class ContributionRequest(
   componentId: Option[String],
   componentType: Option[ComponentType],
   source: Option[AcquisitionSource],
-  abTest: Option[AbTest]
+  abTest: Option[AbTest], // Deprecated
+  refererAbTest: Option[AbTest],
+  nativeAbTests: Option[Set[AbTest]]
```

Comment:
would a better name be `contributionsAbTests`?

Reply:
No, because this field is going to be used by S&C as well. Open to another name though...